File: JavaScript/problem-1.js
const multipleOfX = (n, x) => {
    return n % x === 0;
}

const multipleOf3And5 = (end) => {
    let multiplesSum = 0;

    for(let i = 0; i < end; i++) {
        if(multipleOfX(i, 3) || multipleOfX(i, 5)) {
            multiplesSum = multiplesSum + i;
        }
    }

    return multiplesSum;
}

const result = multipleOf3And5(1000);
console.log(result);

File: Python/problem-1.py
def multiple_of_x(n, x):
    return n % x == 0


def multiple_of_3_and_5(end):
    result = 0
    for i in range(end):
        if multiple_of_x(i, 3) or multiple_of_x(i, 5):
            result = result + i
    return result


print(multiple_of_3_and_5(1000))
